src/main/java/pl/targosz/homework/FileSecondLineWrite.java

```diff
@@ -0,0 +1,22 @@
+package pl.targosz.homework;
+
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+public class FileSecondLineWrite {
+    public static void main(String[] args) throws IOException {
+        Path file = Paths.get("src/main/java/pl/targosz/homework/resources/data.txt");
+        double lineCounter = Files.lines(file).count();
+
+            for (int n=0;n<lineCounter;n+=2) {
+                String line = Files.readAllLines(Paths.get("src/main/java/pl/targosz/homework/resources/data.txt")).get(n);
```

Comment:
Wczytaj sobie ten plik **raz**, a później w pętli wyciągaj sobie co drugą linię. Aktualnie wczytujesz ten plik liczbalini/2 razy czyli np. plik 1gb z 1mld linii będziesz wczytywał 0.5mld razy. To straaaaasznie wolne będzie.